addons/website/static/src/js/tours/tour_utils.js

```diff
@@ -154,6 +155,23 @@ function clickOnEdit(position = "bottom") {
     };
 }
 
+/**
+ * Click on the top right edit button and wait for the edit mode
+ * @param {*} position Where the purple arrow will show up
```

Comment:
Maybe the param could be introduces in a forward port but I think it's ok to already add it in 14.0 :ok_hand: